continuous_integration/travis/runtests.sh

```diff
@@ -27,11 +27,20 @@ if [ "$JOBLIB_TESTS" = "true" ]; then
     (cd $JOBLIB/externals && bash copy_loky.sh $TRAVIS_BUILD_DIR)
     pytest -vl --ignore $JOBLIB/externals --pyargs joblib
 else
-    # Run the tests and collect trace coverage data both in the subprocesses
-    # and its subprocesses.
-    PYTEST_ARGS="-vl --timeout=60 --maxfail=5"
+
+    export PYTEST_ARGS="-vl --timeout=60 --maxfail=5"
+    if [ "$LOKY_TEST_FORCE_OPENBLAS" = "true" ]; then
```

Comment:
Nitpick: this variable name would be easier to understand if it was named `LOKY_TEST_ASSUME_OPENBLAS`.